suzieq/poller/worker/inventory/inventory.py

```diff
@@ -55,7 +55,7 @@ def running_nodes(self) -> List[Coroutine]:
         """
         return self._node_tasks
 
-    async def build_inventory(self) -> Dict[str, Node]:
+    async def build_inventory(self, cmd_sem) -> Dict[str, Node]:
```

Comment:
Could you add the type hint here and in the other functions with the semaphore as arg?